grpc/bcodes_test.go

```diff
@@ -30,11 +30,11 @@ func TestErrors(t *testing.T) {
 	}
 
 	for _, tc := range testcases {
```

Comment:
Need some more tests here. Not only wrap/unwrap, but I'd like to see a test that passes the error types as a trailer through a ChillServer and ensures they get passed through correctly.